src/ICSharpCode.SharpZipLib/GZip/GzipOutputStream.cs

```diff
@@ -104,6 +105,27 @@ public int GetLevel()
 		{
 			return deflater_.GetLevel();
 		}
+
+		//AFONSO DUTRA
```

Comment:
Please don't put your attribution in comments like this.
Git keeps track of who made which changes.